### PR TITLE
WAII-2367: Allow passing db_content_filters while adding db connection

### PR DESCRIPTION
**Pull request by pravin-waii**

### CODE REVIEW

File: tests/database_tests.py [hunk 1]
```diff
@@ -75,6 +76,7 @@ def test_modify_connections(self):
                 break
 
         # then add the new connection
+        self.db_conn
```
Comment (swarajj5534) [inline tests/database_tests.py:79]: whats this?

File: waii_sdk_py/database/database.py [hunk 2]
```diff
@@ -148,6 +148,36 @@ class CatalogDefinition(BaseModel):
...
+    include = "include"
+    exclude = "exclude"
+
+class DBContentFilterActionType(str, Enum):
+    # this means we will include/exclude the content
+    visibility = "visibility"
+
+    # this means we will do sample value or not
+    sample_values = "sample_values"
+
```
Comment (swarajj5534) [inline waii_sdk_py/database/database.py:166]: search context is already defined right?

Comment (pravin-waii) [reply waii_sdk_py/database/database.py:166]: yeah my mistaked removed

File: tests/database_tests.py (hunk 1 above)
Comment (pravin-waii) [reply tests/database_tests.py]: removed

**Pull request merged**